src/services/meta_transaction_service.ts

```diff
@@ -138,8 +138,8 @@ export class MetaTransactionService {
 
         const response: CalculateMetaTransactionPriceResponse = {
             takerAddress,
-            sellAmount,
-            buyAmount,
+            buyAmount: makerAssetAmount,
+            sellAmount: totalTakerAssetAmount,
```

Comment:
Noticed that the metatx endpoint would only return the argument you pass to it, whereas the swap endpoint always returns `buyAmount` and `sellAmount`.